### DISCUSSION TITLE
v1.6.0

### DISCUSSION THREAD

**Discussion by zlinoliver** [tuya/tuya-home-assistant#580] [Announcements]
# Important Note

This release is mainly for code refactoring based on HA Core Tuya PRs: https://github.com/home-assistant/core/pulls?q=tuya

The Tuya v2 integration is finally released on HA platform on October 7th, you can check this link for details: [Tuya HA Core version](https://www.home-assistant.io/integrations/tuya/)

But for now, the Tuya HA core version is not as powerful as the Tuya v2 in this Github repo, we will continue to cooperate with the Home Assistant team to add more products support and fix bugs for the Tuya HA core version. 

Please note that you cannot use Tuya v2 and Tuya HA core version at the same time.

You can get more details from the sheet below:

<img width="646" alt="supported_version" src="https://user-images.githubusercontent.com/907831/137506323-8f476c40-8f3f-43a3-8d64-276fb7af023a.png">

<hr /><em>This discussion was created from the release <a href='https://github.com/tuya/tuya-home-assistant/releases/tag/v1.6.0'>v1.6.0</a>.</em>

**Comment by weitheng** [comment]
@zlinoliver Are there any plans to allow Tuya V2 and Tuya Core Version to run concurrently?
